### ISSUE TITLE
Version update

### ISSUE THREAD

**Issue by melkiy84** [ZcashFoundation/zecwallet#398]
When a new version of a fullnode will be awailable? I recieve notification that this version will be depricated soon.

**Comment by tedp111** [comment]
Yes the current version is now deprecated! Please can you build the latest version. Many thanks!!

**Comment by melkiy84** [comment]
It is amazing, I wrote my post more then week ago, already 2 days full node not working. And there is no reaction at all from dev. team. It would be great to see a reaction on issues from the team. And here I see a full ignor of the comunity from you guys. To tell the truth, I don't even think thet you read this posts :(

**Comment by dandare76** [comment]
Yes, please provide compiled Windows binary and setup! Node is not working!

**Comment by melkiy84** [comment]
I haven't seen such an attitude to users at all. Guys, if you stoped supporting this program, inform us please. I am waiting for update of fullnode, and see no reaction from your team on requests.

**Comment by melkiy84** [comment]
> I have tried to compile the Windows version of the node myself. No .sln file or makefile for Visual C++ exists, so I couldn't do it. All other cryptocurrency nodes are quckly available in Windows versions immediately after a new source code version comes out. In my opinion ZCash is an abandoned project.

The problem is not in zcash, the probem is that developer of zcashd stoped supporting windows. And as I understood Zecwallet was a gui bassed on zcashd wallet itsself. From now on, if you want to use zcashd, you can do it with wsl build in functionality of windows. And the work will be only from console. Right now finissshed configuring all the staff on my pc, and migrated wallet and all data there. It iss the besst solution at this moment for me.

**Comment by solcast13** [comment]
Really need a version update, at least a final update to allow us to grab private keys.  
I cannot get wsl to work on current machine.  
presently stuck with "this version has been deprecated as of block heigh 1879374.  You should upgrade to the lastest version of zcash."  I really just need this shit to work long enough to dump my private keys and move my wallet somewhere else... Is there a way to get it to launch offline ?  Thus far all my efforts have not worked.

**Comment by adityapk00** [comment]
Updated both Zecwallet fullnode and the zcashd binaries. 

https://github.com/ZcashFoundation/zecwallet/release
https://github.com/adityapk00/zcash/releases

**Comment by melkiy84** [comment]
It is great, that you updated versssion. But it was to slow, and I don't see a reason to migrate to windows app. again. Considering that in future such situation can happen again, and my asssets will be blocked from ussage, better to use a linux app.
